src/app/forms/templates/test-records/section-templates/test/desk-based-test-section-group2-PSV.ts

```diff
@@ -0,0 +1,146 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes, FormNodeWidth } from '@forms/services/dynamic-form.types';
+
+export const DeskBasedTestSectionGroup2PSV: FormNode = {
+  name: 'testSection',
+  label: 'Test',
+  type: FormNodeTypes.GROUP,
+  children: [
+    {
+      name: 'testStartTimestamp',
+      label: 'Test start date',
+      type: FormNodeTypes.CONTROL,
+      value: new Date().toISOString(),
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN
+    },
+    {
+      name: 'testEndTimestamp',
+      type: FormNodeTypes.CONTROL,
+      label: 'Test end date',
+      value: new Date().toISOString(),
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN,
+      validators: [{ name: ValidatorNames.AheadOfDate, args: 'testStartTimestamp' }]
+    },
+    {
+      name: 'testTypes',
+      type: FormNodeTypes.ARRAY,
+      children: [
+        {
+          name: '0',
+          type: FormNodeTypes.GROUP,
+          children: [
+            {
+              name: 'testDate',
+              type: FormNodeTypes.CONTROL,
+              label: 'Test Date',
+              value: '',
+              viewType: FormNodeViewTypes.DATE,
+              editType: FormNodeEditTypes.DATE
+            },
+            {
+              name: 'testType',
+              type: FormNodeTypes.CONTROL,
+              label: 'Test Type',
+              value: '403/404',
+              editType: FormNodeEditTypes.TEXT,
+              width: FormNodeWidth.L
+            },
+            {
+              name: 'testCode',
+              type: FormNodeTypes.CONTROL,
+              label: 'Test Code',
+              value: '',
+              width: FormNodeWidth.L
+            },
+            {
+              name: 'testResult',
+              type: FormNodeTypes.CONTROL,
+              label: 'Result',
+              viewType: FormNodeViewTypes.HIDDEN,
+              editType: FormNodeEditTypes.RADIO,
+              options: [
+                { value: 'pass', label: 'Pass' },
+                { value: 'fail', label: 'Fail' }
+              ],
+              required: true
+            },
+            {
+              name: 'reasonForAbandoning',
+              type: FormNodeTypes.CONTROL,
+              value: null,
+              viewType: FormNodeViewTypes.HIDDEN,
+              editType: FormNodeEditTypes.HIDDEN,
+              required: true
+            },
+            {
+              name: 'additionalCommentsForAbandon',
+              type: FormNodeTypes.CONTROL,
+              value: null,
+              viewType: FormNodeViewTypes.HIDDEN,
+              editType: FormNodeEditTypes.HIDDEN,
+              required: true
+            },
+            {
+              name: 'certificateNumber',
+              type: FormNodeTypes.CONTROL,
+              label: 'Certificate number',
+              value: '' || null,
```

Comment:
Pick one. `''` is false so this would always eval to null. Given the validators on this, pick on that is appropriate.